### PR TITLE
#174 Implement least priviledge service accounts for the core functions of the landing zone

### PR DESCRIPTION
**Pull request by davelanglois-ssc**

### CODE REVIEW

**Review by obriensystems — COMMENTED** [review]
cloning branch, reviewing over today/friday

**Comment by davelanglois-ssc** [comment]
please squash commit before merging. I forgot to do it on our fork.

**Comment by cartyc** [comment]
Thanks Dave! Took a look at the pre-merge branch yesterday and it looked great! I will do another pass today to make sure I didn't miss anything. Looks like a great step in the right direction :)

File: solutions/landing-zone-namespaced/landing-zone/setters.yaml
Comment (cartyc) [inline solutions/landing-zone-namespaced/landing-zone/setters.yaml:38]: Are any services planned on being deployed to the default `config-control` namespace? If not we could remove this value.

File: solutions/landing-zone-namespaced/landing-zone/services.yaml
Comment (cartyc) [inline solutions/landing-zone-namespaced/landing-zone/services.yaml:18]: For these services would it be best to put them in a special namespace for the host project or keep them as part of a bootstrap script?

File: solutions/landing-zone-namespaced/landing-zone/setters.yaml
Comment (davelanglois-ssc) [reply solutions/landing-zone-namespaced/landing-zone/setters.yaml:38]: my plan is that the config-control namespace (yakima SA) will be granted org admin privilege and will be used to manage IAM persmissions only. Down the road, I was thinking of creating another management namespace with the org admin privilege and use that one instead of the config-control namespace to avoid granting very high level permission to that yakima SA which comes from an external project

File: solutions/landing-zone-namespaced/landing-zone/services.yaml
Comment (davelanglois-ssc) [reply solutions/landing-zone-namespaced/landing-zone/services.yaml:18]: I vote for the second option,,bootstrap script

File: solutions/landing-zone-namespaced/landing-zone/setters.yaml
Comment (cartyc) [reply solutions/landing-zone-namespaced/landing-zone/setters.yaml:38]: Did you want to leave it as is for now and patch that in a future release?

File: solutions/landing-zone-namespaced/landing-zone/services.yaml
Comment (cartyc) [reply solutions/landing-zone-namespaced/landing-zone/services.yaml:18]: Sounds good to me.

**Review by cartyc — APPROVED** [review]

**Comment by obriensystems** [comment]
Reviewing and deploying - but I don't want to hold up the PR Merge
preliminary visual review very good - but I will need to verify all namespaces config-control, logging, hierarchy, projects, policies and in particular networks
LGTM

**Comment by fmichaelobrien** [comment]
LGTM - deeper review of the networks namespace - post merge

**Pull request merged**
